data/core/tips.json

```diff
@@ -312,16 +312,81 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Survivor saved is a friend earned.  Most of the time..." ]
+    "text": [ "A survivor saved is a friend earned.  Most of the time..." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Learning how to play?  Visit keybindings menu and learn your ropes." ]
+    "text": [ "Learning how to play?  Visit the keybindings menu and learn your ropes." ]
   },
   {
     "type": "snippet",
     "category": "tip",
     "text": [ "Expect the unexpected, even if it's the Spanish Inquisition." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You can throw things while peeking around corners.  Perfect for taking down gun turrets." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Nobody told the vending machines that the world ended.  Save those cash cards!" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There's got to be some cool stuff in those top-secret underground labs, right?" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Put a funnel over a jug or barrel to collect rainwater over time." ]
```

Comment:
It might be a good idea to mention in a separate tip that different funnels have different efficiency. That's not an obvious mechanic, and wouldn't hurt to share.

Reply:
That mechanic is so non-obvious that I didn't even know that was a thing.

Reply:
I also have no idea how I'd mention this in a witty way.